utils/_context/containers.py

```diff
@@ -573,8 +573,10 @@ def __init__(self, name, image_name, host_log_folder, proxy_port, environment) -
         self.interface = None
         self.environment["AWS_ACCESS_KEY_ID"] = os.environ.get("AWS_ACCESS_KEY_ID", "")
```

Comment:
maybe a good idea to have `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` prefixed also ?

Reply:
done